backends/edxorg.py

```diff
@@ -54,14 +59,17 @@ def get_user_details(self, response):
                 the following keys:
                 <remote_id>, `username`, `email`, `fullname`, `first_name`, `last_name`
         """
-        return {
-            'edx_id': 'unique_edx_id_1',
-            'username': 'my_username',
-            'email': 'firstname_lastname@example.com',
-            'fullname': 'firstname lastname',
-            'first_name': 'firstname',
-            'last_name': 'lastname'
+        full, first, last = self.get_user_names(response['name'])
+
+        retval = {
+            'edx_id': response['id'],
```

Comment:
do we want to use this as remote unique identifier? Isn't it better to use the email that will be unique even if there is a change in the DB?